.jscsrc

```diff
@@ -1,3 +1,4 @@
 {
-  "preset": "google"
+  "preset": "google",
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties"
```

Comment:
One argument for this is that properties may come straight from JSON which doesn't obey these rules